pkg/network/tracer/tracer_test.go

```diff
@@ -1881,3 +1883,186 @@ func skipIfWindows(t *testing.T) {
 		t.Skip("test unavailable on windows")
 	}
 }
+
+var (
+	disableTLSVerification = sync.Once{}
+)
+
+const (
+	numberOfRequests      = 100
+	numberOfIterations    = 100
+	pythonSSLServerFormat = `import http.server, ssl
+
+class RequestHandler(http.server.BaseHTTPRequestHandler):
```

Comment:
please move the python server logic in pkg/network/http/testutil/testutil.go with the same API (this would be useful for other test and in http/monitor_test.go)
HTTPPythonServer(t *testing.T, addr string, options Options) func()

Reply:
done